NuGet.config

```diff
@@ -2,11 +2,11 @@
 <configuration>
   <packageSources>
     <clear />
-    <!--
-      Restore sources should be defined in build/sources.props.
-      The only allowed feed here is MSBuild SDK packages which is required to work around
-      https://github.com/Microsoft/msbuild/issues/2914
-    -->
+    <add key="aspnetcore dev" value="https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json" />
```

Comment:
You don't have to address this now, but let's open an item to prune our myget feeds. At this point, I think we shouldn't need any.